File-Formats/PDF/Convert-HTML-To-PDF/features.md

```diff
@@ -1732,6 +1732,74 @@ document.Close(True)
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Initialize HTML to PDF converter.
+HtmlToPdfConverter htmlConverter = new HtmlToPdfConverter();
+BlinkConverterSettings blinkConverterSettings = new BlinkConverterSettings();
+//Set Blink viewport size.
+blinkConverterSettings.ViewPortSize = new Size(1280, 0);
+//Set the html margin top value based on the html header height and margin top value.
+blinkConverterSettings.Margin.Top = 70;
+//Set the html margin bottom value based on the html footer height and margin bottom value.
+blinkConverterSettings.Margin.Bottom = 40;
+//sets the custom HTML header to add at the top of each page.
+blinkConverterSettings.HtmlHeader = File.ReadAllText("header.html"); 
+//sets the custom HTML footer to add at the bottom of each page.
+blinkConverterSettings.HtmlFooter = File.ReadAllText("footer.html");
+//Assign Blink converter settings to HTML converter.
+htmlConverter.ConverterSettings = blinkConverterSettings;
+//Convert URL to PDF document.
+PdfDocument document = htmlConverter.Convert("https://www.syncfusion.com");
+
+//Create a filestream.
+FileStream fileStream = new FileStream("Output.pdf", FileMode.Create, FileAccess.ReadWrite);
+//Save and close the PDF document.
+document.Save(fileStream);
+document.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Initialize the HTML to PDF converter.
+Dim htmlConverter As HtmlToPdfConverter = New HtmlToPdfConverter()
+Dim blinkConverterSettings As BlinkConverterSettings = New BlinkConverterSettings()
+'Set Blink viewport size.
+blinkConverterSettings.ViewPortSize = New Size(1280, 0)
+'Set the html margin top value based on the html header height and margin top value.
+blinkConverterSettings.Margin.Top = 70
+'Set the html margin bottom value based on the html footer height and margin bottom value.
+blinkConverterSettings.Margin.Bottom = 40
+'sets the custom HTML header to add at the top of each page.
+blinkConverterSettings.HtmlHeader = File.ReadAllText("header.html")
+'sets the custom HTML footer to add at the bottom of each page.
+blinkConverterSettings.HtmlFooter = File.ReadAllText("footer.html")
+'Assign Blink converter settings to HTML converter.
+htmlConverter.ConverterSettings = blinkConverterSettings
+'Convert URL to PDF document.
+Dim document As PdfDocument = htmlConverter.Convert("https://www.syncfusion.com")
+
+'Create file stream to save the PDF document. 
+Dim fileStream As FileStream = New FileStream("Output.pdf", FileMode.CreateNew, FileAccess.ReadWrite)
+'Save and close the PDF document.
+document.Save(fileStream)
+document.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from GitHub.
+
+N> Need to set the html margin top and bottom value based on the html header footer height and margin value.
```

Comment:
Adjust the HTML margin top and bottom values according to the height of the HTML header and footer, as well as the margin settings.

Reply:
changed